child_compassion/demo/compassion.child.csv

```diff
@@ -0,0 +1,67 @@
+id,code,name
+child_01,"BR2150576","Marilyn Monroe"
+child_02,"RW1440285","Abraham Lincoln"
```

Comment:
You could speed up install by using children of same projects (does not need to fetch data if already exists).
For those who are in the same country (i.e. RW144XXXX, RW530XXXX) you can change and put only RW144XXXX (one project per country).